### PR TITLE
Feature/862 render suggestions google

### LINKED ISSUE #862: Render Suggestions from Google
```
{
   "conversation_id": "UUID",
   "message":{
    "text": "Hello, world"',
    "fallback": "Hello, world!\n\nReply with \"Hello\" or \"Hi!\"",
    "suggestions": [
      {
        "reply": {
          "text": "Hello",
          "postbackData": "hello-formal",
        },
      },
      {
        "reply": {
          "text": 'Hi!',
          "postbackData": "hello-informal",
        }
      }
    ]
  }
}
```

### PR DESCRIPTION
**Pull request by AudreyKj**
closes #862 

changes: 
- Google render
- Suggestions component 
- removed time and avatar from Image component (as the extraction has already been merged) + added altText

styling: follows Google's styling on GBM chat

useful docs: 
[Google docs - Receive message]( https://developers.google.com/business-communications/business-messages/guides/build/receive)
[Google docs - Send messages](https://developers.google.com/business-communications/business-messages/guides/build/send)
[Google docs - SuggestedReply](https://developers.google.com/business-communications/business-messages/reference/rest/v1/conversations.messages#SuggestedReply)

screenshots: 
Screenshot 2021-02-19 at 15 55 49
Screenshot 2021-02-19 at 15 55 36
Screenshot 2021-02-19 at 15 53 42
Screenshot 2021-02-19 at 16 22 23
Screenshot 2021-02-19 at 16 26 07

Note: a hover text (grey background) has been added for actions that can only be triggered by the contact (contact a live agent, call, or authenticate with Google).

Screenshot 2021-02-19 at 16 04 44

### CODE REVIEW

File: lib/typescript/render/providers/google/components/Suggestions/index.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                return (
+                  <button key={Math.floor(Math.random() * 50)} className={styles.replyButton}>
+                    <h1 key={Math.floor(Math.random() * 50)} className={styles.title}>
+                      Message a live agent
+                    </h1>
+                  </button>
+                );
+              }
+            })}
+            {suggestions &&
```
Comment (AudreyKj) [inline lib/typescript/render/providers/google/components/Suggestions/index.tsx:103]: grey hover text appears on actions that can only be triggered by the contact (but I kept the external links clickable)

**Review by AitorAlgorta — APPROVED** [review]
Looks good, but this doesn't fix the sending from the UI right?

File: lib/typescript/render/components/Image/index.tsx [hunk 2]
```diff
@@ -1,32 +1,28 @@
...
+            <img className={styles.messageListItemImageBlock} src={imageUrl} alt={altText ?? null} />
           </div>
-          {sentAt && <div className={styles.messageTime}>{sentAt}</div>}
         </div>
       ) : (
         <div className={styles.messageListUserContainer}>
-          <div className={styles.messageAvatar}>
-            <Avatar contact={contact} />
-          </div>
+          <div className={styles.messageAvatar}></div>
```
Comment (AitorAlgorta) [inline lib/typescript/render/components/Image/index.tsx:21]: do we need this?

File: lib/typescript/render/providers/google/GoogleRender.tsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    type: 'text',
+    text: 'Unkown message type',
+  };
+}
+
+function googleOutbound(message: Message): ContentUnion {
+  const messageJson = message.content;
+
+  if (messageJson.suggestions) {
+    if (messageJson.suggestions.length > 13) {
```
Comment (AitorAlgorta) [inline lib/typescript/render/providers/google/GoogleRender.tsx:91]: Maybe create a constant like `MAX_NUMBER_OF_CARDS` or something like that?

**Review by AitorAlgorta — APPROVED** [review]
LGTM, just 2 small comments

File: lib/typescript/render/components/Image/index.tsx (hunk 2 above)
Comment (AudreyKj) [reply lib/typescript/render/components/Image/index.tsx]: good catch!

File: lib/typescript/render/providers/google/components/Suggestions/index.tsx [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+type SuggestionsRendererProps = DefaultMessageRenderingProps & {
+  text?: string;
+  fallback?: string;
+  image?: {
+    fileUrl: string;
+    altText: string;
+  };
+  suggestions:
+    | SuggestedReplies[]
```
Comment (bitboxer) [inline lib/typescript/render/providers/google/components/Suggestions/index.tsx:23]: Maybe you could add a combined type in the `googleModel` ? that way you don't have to duplicate this in line 44-47 and in the model file itself?

**Review by bitboxer — APPROVED** [review]
💚  . One little nitpick, otherwise great stuff

**Review by bitboxer — APPROVED** [review]
Awesome!

**Pull request merged**